torch/csrc/jit/mobile/register_mobile_autograd.cpp

```diff
@@ -71,7 +71,7 @@ void conv2d_kernel(const c10::OperatorHandle& op, Stack* stack) {
       "conv2d",
       std::vector<c10::IValue>({input, weight, bias}),
       at::sequence_number::peek());
-  auto result_ = VariableType::conv2d(
+  auto result_ = at::TypeDefault::conv2d(
```

Comment:
Can we simply call "at::conv2d" here? cc: @iseeyuan

Reply:
The autograd registration was unified to codegen internally. I'm planning to remove it from OSS. So it does not matter much.